### PR TITLE
KFSPTS-20549 pull GeneralLedgerPendingEntryServiceImpl.java from 10-29 to fix poster, merged to master

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

**Comment by jhulslander** [comment]
Dave gave the sign off to move to test, but I didn't resolve his sub task, I think this can be merged

File: src/main/java/org/kuali/kfs/sys/service/impl/GeneralLedgerPendingEntryServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,1081 @@
...
+                        cashOffsetObject.getFinancialObjectCode(), bank.getCashOffsetSubObjectCode());
+                return false;
+            }
+
+            bankOffsetEntry.setFinancialSubObjectCode(bank.getCashOffsetSubObjectCode());
+        }
+
+        bankOffsetEntry.setTransactionEntryOffsetIndicator(true);
+        bankOffsetEntry.setTransactionLedgerEntryAmount(depositAmount.abs());
+        bankOffsetEntry.setUniversityFiscalPeriodCode(financialDocument.getPostingPeriodCode());
```
Comment (cah292) [inline src/main/java/org/kuali/kfs/sys/service/impl/GeneralLedgerPendingEntryServiceImpl.java:754]: Does this line also need to be updated to set the fiscal period to null? I believe this line had the null-value setup prior to our 10/29/2020 financials upgrade.

**Review by cah292 — COMMENTED** [review]
I had a question about whether an additional line needs to be updated. Other than that, the changes look good.

File: src/main/java/org/kuali/kfs/sys/service/impl/GeneralLedgerPendingEntryServiceImpl.java (hunk 1 above)
Comment (jhulslander) [reply src/main/java/org/kuali/kfs/sys/service/impl/GeneralLedgerPendingEntryServiceImpl.java:754]: This line wasn't changed in recent history, only the line were I commented the CU customization was changed.  Also, the kfs-dev validation only validated that one change.  I don't think we should add changes.

Comment (jhulslander) [reply src/main/java/org/kuali/kfs/sys/service/impl/GeneralLedgerPendingEntryServiceImpl.java:754]: Yes, I double checked the kfs-dev version, this line exists there, and testing didn't find a problem

Comment (cah292) [reply src/main/java/org/kuali/kfs/sys/service/impl/GeneralLedgerPendingEntryServiceImpl.java:754]: Ok, thanks for checking. I'm fine with moving this change forward as-is, but we'll need the functionals to verify that the bank offset entries are being posted to the correct fiscal period. We may need a follow-up PR if this line does end up needing a change.

Comment (jhulslander) [reply src/main/java/org/kuali/kfs/sys/service/impl/GeneralLedgerPendingEntryServiceImpl.java:754]: agreed

**Review by cah292 — APPROVED** [review]
Changes look good, unit tests pass, the user story has sub-tasks for preventing merge conflicts when the develop version of the fix moves to master, and my concerns have been addressed for the moment. Merging.

**Pull request merged**